src/ast-writer.cc

```diff
@@ -69,6 +71,8 @@ struct Context {
   int func_type_index;
 };
 
+}  // namespace
```

Comment:
Could do, shouldn't really make a difference either way. Is there a benefit to not having it nested?